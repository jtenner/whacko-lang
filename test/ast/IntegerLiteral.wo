export fn _start(): void {
  1;
}