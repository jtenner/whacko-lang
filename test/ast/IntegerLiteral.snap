{
  '$type': 'Program',
  declarations: [
    {
      '$type': 'FunctionDeclaration',
      returnType: { '$type': 'ID', name: 'void', typeParameters: [ [length]: 0 ] },
      name: {
        '$type': 'ID',
        name: '_start',
        typeParameters: [ [length]: 0 ]
      },
      export: true,
      block: {
        '$type': 'BlockStatement',
        statements: [
          {
            '$type': 'ExpressionStatement',
            expression: { '$type': 'IntegerLiteral', value: '1' }
          },
          [length]: 1
        ]
      },
      decorators: [ [length]: 0 ],
      parameters: [ [length]: 0 ],
      typeParameters: [ [length]: 0 ]
    },
    [length]: 1
  ],
  exports: [ [length]: 0 ],
  imports: [ [length]: 0 ],
  [Symbol(fullPath)]: 'test/ast/IntegerLiteral.wo'
}